src/sentry/receivers/outbox/control.py

```diff
@@ -162,15 +162,17 @@ def process_relocation_reply_with_export(payload: Mapping[str, Any], **kwds):
     relocation_storage = get_relocation_storage()
     path = f"runs/{uuid}/saas_to_saas_export/{slug}.tar"
     try:
-        encrypted_contents = relocation_storage.open(path)
+        encrypted_bytes = relocation_storage.open(path)
```

Comment:
What happens if you do something like relocation_sorage.open(path, encoding="utf-8") ? This seems like a potentially risky change

Reply:
It will fail, which is fine, because the file we're storing is not a string? It's an encrypted tarball - opening it incorrectly will result in it not opening.

Reply:
ah ok, so we need to ensure that the bytes are preserved as is in order to be decrypted later